lib/sufia/engine.rb

```diff
@@ -23,6 +23,9 @@ class Engine < ::Rails::Engine
     initializer 'requires' do
       require 'activerecord-import'
       require 'hydra/derivatives'
+      require 'sufia/analytics'
```

Comment:
Everything, as in, even stuff in random subdirectories? I thought that was true too but in https://github.com/projecthydra/sufia/blob/master/lib/sufia/engine.rb#L8-L12 we have `app/models/concerns` listed. Could be cruft of course.